src/index/base.cpp

```diff
@@ -205,6 +191,20 @@ void BaseIndex::ThreadSync()
                            __func__, pindex->GetBlockHash().ToString());
                 return;
             }
+
+            auto current_time{std::chrono::steady_clock::now()};
+            if (last_log_time + SYNC_LOG_INTERVAL < current_time) {
+                LogPrintf("Syncing %s with block chain from height %d\n",
```

Comment:
I agree on the change with a small nuance: now the "syncing index from height <num>" is not entirely accurate. When the log is triggered, the block presented here was already scanned.